channels/hr.m3u

```diff
@@ -17,9 +17,9 @@ http://195.29.70.89/PLTV/88888888/224/3221226281/index.m3u8
 http://195.29.70.67/PLTV/88888888/224/3221226027/index.m3u8
 #EXTINF:-1 tvg-id="MAXtvPromoKanal.hr" tvg-name="MAXtv Promo Kanal" tvg-country="HR" tvg-language="Croatian" tvg-logo="https://i.imgur.com/RApnwWl.jpg" group-title="",MAXtv Promo Kanal (576p) [NO KODI]
 http://195.29.70.84/PLTV/88888888/224/3221226027/index.m3u8
-#EXTINF:-1 tvg-id="NewsBar.hr" tvg-name="NewsBar" tvg-country="HR" tvg-language="Croatian" tvg-logo="https://i.imgur.com/Q8eH5zN.jpg" group-title="",NewsBar (528p) [NO KODI]
+#EXTINF:-1 tvg-id="NewsBar.hr" tvg-name="NewsBar" tvg-country="HR" tvg-language="Croatian" tvg-logo="https://i.imgur.com/Q8eH5zN.jpg" group-title="News",NewsBar (528p) [NO KODI]
 http://195.29.70.67/PLTV/88888888/224/3221226407/index.m3u8
-#EXTINF:-1 tvg-id="NewsBar.hr" tvg-name="NewsBar" tvg-country="HR" tvg-language="Croatian" tvg-logo="https://i.imgur.com/Q8eH5zN.jpg" group-title="",NewsBar (528p) [NO KODI]
+#EXTINF:-1 tvg-id="NewsBar.hr" tvg-name="NewsBar" tvg-country="HR" tvg-language="Croatian" tvg-logo="https://i.imgur.com/Q8eH5zN.jpg" group-title="News",NewsBar (528p) [NO KODI]
```

Comment:
I think it was @Sphinxroot that added the "[NO KODI]" tags. What's up with those ?